front-end-admin/src/components/need/NeedInfo.vue

```diff
@@ -3,37 +3,47 @@
     <!-- Main jumbotron for a primary marketing message or call to action -->
     <div class="jumbotron">
        <div class="container">
-        <h2>Need Info Page for {{ need.name }}</h2>
-        <form v-on:submit.prevent="onFormSubmit" class="form-horizontal" >
-          <button type="submit">Update Need</button>
-          <input type="hidden" name="need_id" v-model="need.id">
-          <div class="form-group">
-            <label class="control-label col-sm-2">Full Name:</label>
-            <input name="need_name" type="text" width="100" v-model="need.name">
-          </div>
-          <div class="form-group">
-            <label class="control-label col-sm-2">Description:</label>
-            <textarea rows="4" cols="50" v-model="need.description" form="needform">need_description</textarea>
-          </div>
-          <div class="form-group">
-            <label class="control-label col-sm-2">Need URL:</label>
-            <input name="need_url" type="text" width="50" v-model="need.url">
-          </div>
-          <!-- <div class="form-group">
-            <label class="control-label col-sm-2">Quantity:</label>
-            <input id="need_quantity" name="need_quantity" type="number" min="1" step="1" v-model="need_quantity"/>
-          </div> -->
-          <div class="form-group">
-            <label class="control-label col-sm-2">Units:</label>
-            <input name="need_unit" type="text" width="50" v-model="need.unit">
-          </div>
-          <!-- <div class="form-group">
-            <label class="control-label col-sm-4>">Status: </label>
-            <input type="checkbox" name="active" checked="{{active}}">Active?
-          </div> -->
-            <!-- <button type="submit">Update Need</button> -->
-          <!-- </div> -->
-        </form>
+         <br>
+         <br>
+         <b-card :title="need.name">
```

Comment:
Convert the b-form-textareas that only have 1 line or row on this form to be b-form-inputs